docs/config.rst

```diff
@@ -937,6 +966,38 @@ Marks
    Mark used to indicate that the prompt is ready for user input, unless some
    other context overrides it, like a VCS repository.
 
+.. attribute:: LP_MARK_DEV_CLOSE
+   :type: string
+   :value: ">"
+
+   Closing of the "development tools" section.
+
+   .. versionadded:: 2.2
+
+.. attribute:: LP_MARK_CMAKE
```

Comment:
This block still not alphabetical.

Reply:
:man_facepalming:

Reply:
To be fair, this isn't something you should need to worry about. Or even me for that matter. This should be something that is checked by the docs linter. Shouldn't be that hard for me to throw something together...